help/destinations/catalog/data-partners/merkury-enterprise-identity.md

```diff
@@ -0,0 +1,168 @@
+
+## Supported identities {#supported-identities}
+
+|Target Identity|Description|Considerations|
+|---|---|---|
+|GAID|Google Advertising ID|Select the GAID target identity when your source identity is a GAID namespace.|
+|IDFA|Apple ID for Advertisers|Select the IDFA target identity when your source identity is an IDFA namespace.|
+|ECID|Experience Cloud ID|A namespace that represents ECID. This namespace can also be referred to by the following aliases: "Adobe Marketing Cloud ID", "Adobe Experience Cloud ID", "Adobe Experience Platform ID". See the following document on [ECID](/help/identity-service/ecid.md) for more information.|
+|phone_sha256|Phone numbers hashed with the SHA256 algorithm|Both plain text and SHA256 hashed phone numbers are supported by Adobe Experience Platform. When your source field contains unhashed attributes, check the **[!UICONTROL Apply transformation]** option, to have [!DNL Platform] automatically hash the data on activation.|
+|email_lc_sha256|Email addresses hashed with the SHA256 algorithm|Both plain text and SHA256 hashed email addresses are supported by Adobe Experience Platform. When your source field contains unhashed attributes, check the **[!UICONTROL Apply transformation]** option, to have [!DNL Platform] automatically hash the data on activation.|
+|extern_id|Custom user IDs|Select this target identity when your source identity is a custom namespace.|
+
+{style="table-layout:auto"}
+
+## Supported audiences
+
+This section describes which type of audiences you can export to this destination.
+
+|**Audience**|**Supported**|**Description**|**origin**|                      
+|---|---|---|---|
+|Segmentation Service|✓|Audiences generated through the Experience Platform [[Segmentation Service]{.underline}](https://experienceleague.adobe.com/en/docs/experience-platform/segmentation/home).|
+|Custom uploads|x|Audiences [[imported]{.underline}](https://experienceleague.adobe.com/en/docs/experience-platform/segmentation/ui/overview#import-audience) into Experience Platform from CSV files.|
+
+{style="table-layout:auto"}
+
+## Export type and frequency
+
+Refer to the table below for information about the destination export type and frequency.
+|**Audience**|**Supported**|**Description origin**|            
+|---|---|---|      
+|Segmentation Service|✓|Audiences generated through the Experience Platform [[Segmentation Service]{.underline}](https://experienceleague.adobe.com/en/docs/experience-platform/segmentation/home).|
+Custom uploads|X|Audiences [[imported]{.underline}](https://experienceleague.adobe.com/en/docs/experience-platform/segmentation/ui/overview#import-audience) into Experience Platform from CSV files.
+
+{style="table-layout:auto"}
+
+## Connect to the destination
+
+>[!IMPORTANT]
```

Comment:
incorrect note formatting, please update

Reply:
@vgiurgiu, the call out here is for a [!IMPORTANT], I have updated the one [!NOTE] item to match the [!IMPORTANT] changes.  Let me know if this is what you were referring to or not.

Like so:
>[!NOTE]

>When selecting the CSV option....

Reply:
hey @framingeinstein , it's fine to leave it as IMPORTANT, my formatting comment relates to the way you use the chevrons. The formatting should be:

```
>[!IMPORTANT]
>
>When selecting the CSV option....
```